checkov/cloudformation/runner.py

```diff
@@ -93,8 +98,8 @@ def get_graph_checks_report(self, root_folder, runner_filter: RunnerFilter):
             for check_result in check_results:
                 entity = check_result['entity']
                 entity_file_abs_path = create_file_abs_path(root_folder, entity.get(CustomAttributes.FILE_PATH))
-                entity_name = entity.get(CustomAttributes.BLOCK_NAME).split(".")[1]
-                entity_context = self.context[entity_file_abs_path][CloudformationTemplateSections.RESOURCES][entity_name]
+                entity_name = entity.get(CustomAttributes.BLOCK_NAME)
+                entity_context = self.context.get(entity_file_abs_path).get(CloudformationTemplateSections.RESOURCES).get(entity_name)
```

Comment:
how it was before, is better, because if it fails you know exactly, that a specific key was not present, now you would get errors telling you that `NoneType` doesn't have a method `get()`